### PR TITLE
fix generate assets

### PR DESCRIPTION
**Pull request by baurine**
What did:

- Fix `scripts/generate_assets.go` crash when building by `DASHBOARD=COMPILE make pd-server` in pd repo

### CODE REVIEW

File: scripts/embed_ui_assets.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ fi
 echo "+ Embed UI assets"
 
 cd "$PROJECT_DIR/scripts"
-go run generate_assets.go "$PROJECT_DIR/ui/build" $BUILD_TAG_PARAMETER
+go run generate_assets.go "$PROJECT_DIR/ui/build" $BUILD_TAG_PARAMETER
```
Comment (baurine) [inline scripts/embed_ui_assets.sh:35]: `$BUILD_TAG_PARAMETER` may be empty

File: scripts/generate_assets.go [hunk 2]
```diff
@@ -16,7 +16,10 @@ func main() {
 		log.Fatal("Require 2 args")
 	}
 	directory := os.Args[1]
-	buildTag := os.Args[2]
+	buildTag := ""
+	if len(os.Args) > 2 {
+		buildTag = os.Args[2]
+	}
```
Comment (baurine) [inline scripts/generate_assets.go:22]: When `$BUILD_TAG_PARAMETER` is empty, `len(os.Args) == 2`, and `os.Args[2]` would crash.

Comment (breeswish) [reply scripts/generate_assets.go:22]: Could you try with `"$BUILD_TAG_PARAMETER"` instead of `$BUILD_TAG_PARAMETER`?

Comment (baurine) [reply scripts/generate_assets.go:22]: Yep, that's would be better, let me try it.

File: scripts/generate_assets.go [hunk 3]
```diff
@@ -12,7 +12,7 @@ import (
 )
 
 func main() {
-	if len(os.Args) < 2 {
+	if len(os.Args) < 3 {
```
Comment (baurine) [inline scripts/generate_assets.go:15]: `os.Args[0]` is the command bin self.

**Review by breeswish — APPROVED** [review]

**Pull request merged**